boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059.rst

```diff
@@ -0,0 +1,142 @@
+.. _nrf52840_pca10059:
+
+nRF52840-PCA10059
+#################
+
+Overview
+********
+
+Zephyr applications use the nrf52840_pca10059 board configuration
+to run on the nRF52840 PCA10059 (USB Dongle) hardware. It provides
+support for the Nordic Semiconductor nRF52840 ARM Cortex-M4F CPU and
+the following devices:
+
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* UART
+* GPIO
+* FLASH
+* RADIO (Bluetooth Low Energy)
+* Segger RTT (RTT Console)
```

Comment:
By default the board does not have any headers or directly connected Segger RTT support, so surely this should be removed or have some descrription of what additional hardware is requires and how to hook it up?

Reply:
Agreed, this should be amended to reflect this fact.